samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -345,6 +348,19 @@ def _clean_running_containers_and_related_resources(self) -> None:
         cast(WarmLambdaRuntime, self.lambda_runtime).clean_running_containers_and_related_resources()
         cast(RefreshableSamFunctionProvider, self._function_provider).stop_observer()
 
+    def _add_account_id_to_global(self, parameters) -> Dict|None:
```

Comment:
We probably don't need to pass in the parameters since we have access to them through `self`.